addon/components/o-s-s/email-input.ts

```diff
@@ -44,17 +44,24 @@ export default class OSSEmailInput extends Component<OSSEmailInputArgs> {
     return this.args.errorMessage || this.regexError || null;
   }
 
+  get onChangeHandler() {
+    return (value: string | null) => this.validateInput(value);
+  }
+
   @action
-  validateInput(): void {
-    this.regexError = '';
-    this.args.onChange?.(this.args.value ? this.args.value.toLowerCase() : this.args.value);
+  validateInput(value: string | null): void {
```

Comment:
I'm not sure I understand these changes 🤔 
Why do we need to use a getter for the validateInput ?
and why is the regexError reset logic moved ?